release.nix

```diff
@@ -69,6 +69,7 @@ in lib.fix (jobsets: testJobsets // {
       ++ (allJobs jobsets.docs) 
       ++ (allJobs jobsets.plutus-playground)
       ++ (allJobs jobsets.meadow)
+      ++ (allJobs jobsets.haskellPackages.deployment-server)
```

Comment:
Shouldn't be necessary - add it to the list of local packages in `lib.nix` and it'll be included automatically.

Reply:
will do